website/docs: air-gapped: Modernize for 2025.8 and address suggestions (#15305)
goauthentik/authentik: website/docs/install-config/air-gapped.mdx

title: Air-gapped environments
---

import TabItem from "@theme/TabItem";
import Tabs from "@theme/Tabs";

## Outbound connections

By default, authentik creates outbound connections to the following URLs:

- https://version.goauthentik.io: Periodic update check

> - https://goauthentik.io: Anonymous analytics on startup.

Author: ```suggestion
- https://goauthentik.io: Anonymous analytics on startup
```

> - https://secure.gravatar.com: Avatars for users.

Author: ```suggestion
- https://secure.gravatar.com: Avatars for users
```

> - https://authentik.error-reporting.a7k.io: Error reporting.

Author: ```suggestion
- https://authentik.error-reporting.a7k.io: Error reporting
```

> - https://tile.openstreetmap.org: Map tiles for event logs :ak-enterprise :ak-version[2025.8].

Author: ```suggestion
- https://tile.openstreetmap.org: Map tiles for event logs :ak-enterprise :ak-version[2025.8]
```

- https://goauthentik.io: Anonymous analytics on startup
- https://secure.gravatar.com: Avatars for users
- https://authentik.error-reporting.a7k.io: Error reporting

> - https://tile.openstreetmap.org: Map tiles for event logs :ak-enterprise :ak-version[2025.8]

Author: Hmm I would assume so, yes. Might as well update this from the old design

  </TabItem>
</Tabs>

Additionally, adjust the following [System settings](../sys-mgmt/settings.md):

> - **Avatars**: By default this setting uses [Gravatar](https://secure.gravatar.com/). Set it to a combination of other options, such as `initials`

Reviewer: ```suggestion
- **Avatars**: By default this setting connects to [Gravatar](https://secure.gravatar.com/). To avoid outgoing connections, set this to a combination of other options, such as `initials`.
```

## Required resources

### Container images

> authentik deployments require access to the following container images. In an air-gapped environment, this can be achieved by mirroring the containers to an internal registry, or using other methods appropriate for your environment.

Author: ```suggestion
authentik deployments require access to the following container images. In an air-gapped environment, this can be achieved by mirroring the images to an internal registry, or using other methods appropriate for your environment.
```

Small fix from Dewi's suggestion as you'd mirror the image not the container

#### Main image

> - `ghcr.io/goauthentik/server`

Author: ```suggestion
- `ghcr.io/goauthentik/server` or `authentik/server`
```

#### Outpost images

> - `ghcr.io/goauthentik/ldap`

Author: ```suggestion
- `ghcr.io/goauthentik/ldap` or `authentik/ldap`
```

> - `ghcr.io/goauthentik/proxy`

Author: ```suggestion
- `ghcr.io/goauthentik/proxy`  or `authentik/proxy`
```

> - `ghcr.io/goauthentik/rac`

Author: ```suggestion
- `ghcr.io/goauthentik/rac` or `authentik/rac`
```

> - `ghcr.io/goauthentik/radius`

Author: ```suggestion
- `ghcr.io/goauthentik/radius` or `authentik/radius`
```

Each outpost type requires specific ports:

> Each outpost container requires access to communicate with the authentik server via whichever protocol is specified in the URL set in the `AUTHENTIK_HOST` environment variable (preferably HTTPS).

Reviewer 2: ```suggestion
Each outpost container, in order to communicate with authentik, requires access to the authentik server via whichever protocol is specified in the URL set in the `AUTHENTIK_HOST` environment variable (preferably HTTPS).
```

> The outpost containers also need certain ports exposed:

Reviewer 2: Ah never mind, I read more about EXPOSED. It's good as is.
